Sources/Bot/Models/RequiredStatusCheck.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+public struct RequiredStatusChecks: Decodable, Equatable {
+    public let strict: Bool // Require branches to be up to date before merging?
```

Comment:
these comments would be better as proper doc comments as these properties are public.